backoffice/views/courses_views.py

```diff
@@ -187,45 +187,42 @@ def enrolled_users(request, course_key_string):
 
 @group_required('fun_backoffice')
 def users_without_proctoru_reservation(request, course_key_string):
-        if not INSTALLED_PU:
-            return HttpResponse('ProctorU xblock not installed')
+    if not INSTALLED_PU:
+        return HttpResponse('ProctorU xblock not installed')
+
+    # what if GET is used?
+    if request.method == 'POST':  # export as CSV
```

Comment:
`# what if GET is used?`
je sais pas, mais on peut mettre un décorateur qui l'empêche: 
`from django.views.decorators.http import require_POST`